docs/classifier_example/classification_v3_4_data_points.ipynb

```diff
@@ -0,0 +1,793 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center> A Quantum distance-based classifier </center>#\n",
+    "## <center> Robert Wezeman, TNO </center> ##"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"contents\"></a>\n",
+    "# Table of Contents\n",
+    "* [Introduction](#introduction)\n",
+    "* [Problem](#problem)\n",
+    "* [Theory](#theory)\n",
+    "* [Algorithm](#algorithm)\n",
+    "* [Implementation](#implementation)\n",
+    "* [Conclusion and further work](#conclusion)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$ \\newcommand{\\ket}[1]{\\left|{#1}\\right\\rangle} $$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Import external python file\n",
+    "import nbimporter\n",
+    "from data_preprocessing_class import DataPlotter  # for easier plotting \n",
+    "\n",
+    "# Import math functions \n",
+    "from math import acos, pi\n",
+    "from numpy import sign\n",
+    "get_bin = lambda x, n: format(int(x), 'b').zfill(n)\n",
```

Comment:
Not very pep8 to assign a name to a lambda function.

Reply:
Removed it, not used in this note book